contracts/optimistic-ethereum/OVM/precompiles/OVM_SequencerEntrypoint.sol

```diff
@@ -30,6 +30,7 @@ contract OVM_SequencerEntrypoint {
      * calldata[00:01]: transaction type (0 == EIP 155, 2 == Eth Sign Message)
      * calldata[01:33]: signature "r" parameter
      * calldata[33:65]: signature "s" parameter
+     * calldata[65:66]: signature "v" parameter
```

Comment:
This is [a ticket](https://github.com/ethereum-optimism/roadmap/issues/273), can we actually not do this here so that there is a unique/standalone PR to give auditors? <3

Reply:
lolol yes